Type not resolved in SelectPanel2/SelectPanel.tsx

### Description

The folder `src` appears in the following imports:

https://github.com/fcollonval/react/blob/c2809d677a44686573f0bd31fc1988e73ef80b84/src/drafts/SelectPanel2/SelectPanel.tsx#L18-L19

In the published package (36.4.0), typescript fails to resolve the types because there is no `src` folder.

### Steps to reproduce

Add latest @primer react within a project
Using VS code the parser flags
`node_modules/@primer/react/lib-esm/drafts/SelectPanel2/SelectPanel.d.ts` with error

```
Cannot find module '../../../src/index' or its corresponding type declarations.ts(2307)
```

### Version

v36.4.0

### Browser

_No response_

--- a/src/drafts/SelectPanel2/SelectPanel.tsx
+++ b/src/drafts/SelectPanel2/SelectPanel.tsx
@@ -15,8 +15,8 @@ import {
   Text,
   ActionListProps,
   Octicon,
-} from '../../../src/index'
-import {ActionListContainerContext} from '../../../src/ActionList/ActionListContainerContext'
+} from '../../index'
+import {ActionListContainerContext} from '../../ActionList/ActionListContainerContext'
 import {useSlots} from '../../hooks/useSlots'
 import {useProvidedRefOrCreate, useId, useAnchoredPosition} from '../../hooks'
 import {useFocusZone} from '../../hooks/useFocusZone'

